atomic_reactor/util.py

```diff
@@ -542,7 +542,7 @@ def print_version_of_tools():
     ('release', 'Release'),
     ('architecture', 'Architecture'),
     ('vendor', 'Vendor'),
-    ('authoritative-source', 'Authoritative_Registry'),
+    ('authoritative-source-url', 'Authoritative_Registry'),
```

Comment:
well it is used for get_preffered_label method, which is used in different modules, explicitly is never called with authoritative-source-url, but in one case it is using some variable and I didn't look on that module yet "post_tag_by_labels.py", but I think this should be complete list of new & old labels